results/cmd/api/main_test.go

```diff
@@ -63,3 +63,33 @@ func setupTestDB(dbName string, t *testing.T) (*sql.DB, error) {
 	}
 	return db, nil
 }
+
+func TestGetTaskRun(t *testing.T) {
+	// Create a temporay database
+	testDB, err := setupTestDB("testdb", t)
+	if err != nil {
+		t.Fatalf("failed to create temp file for db: %v", err)
+	}
+
+	// connect to fake server and do testing
+	srv := &server{db: testDB}
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	objectmeta := &pb.ObjectMeta{Uid: "31415926", Name: "mytaskrun", Namespace: "default"}
```

Comment:
This isn't being reused, so you can just define this in the TaskRun below.

I'd also recommend breaking up the struct to 1 field per line for ease of reading.

Reply:
done